### DISCUSSION TITLE
Rustlings Web Browser Version

### DISCUSSION THREAD

**Discussion by jgarte** [rust-lang/rustlings#2171] [Q&A]
Hi, what do you think if we had a version of Rustlings that we can run in the browser? Not unlike that Go tutorial that runs in the browser.

**Comment by mo8it** [comment]
This doesn't fit the way Rustlings works. We would need an editor with Rust-Analyzer and a terminal in the browser. It would have too many drawbacks.
By installing Rustlings on your system, you get the Rust toolchain and can start your own projects very easily after finishing Rustlings.
